src/Diagnostics/src/LogFile.cpp

```diff
@@ -0,0 +1,56 @@
+#include "OpenLoco/Diagnostics/LogFile.h"
+#include <fmt/chrono.h>
+#include <fmt/format.h>
+#include <fmt/ostream.h>
+
+namespace OpenLoco::Diagnostics::Logging
+{
+    LogFile::LogFile(const fs::path& file)
+    {
+        // Ensure the directory exists in case the filepath is a relative path and contains a sub directory.
+        fs::create_directories(file.parent_path());
+
+        _file.open(file, std::ios::binary | std::ios::app);
+    }
+
+    void LogFile::print(Level level, std::string_view message)
+    {
+        if (!passesLevelFilter(level))
+        {
+            return;
+        }
+
+        if (!_file.is_open())
+        {
+            return;
+        }
+
+        std::string timestamp;
+        if (getWriteTimestamps())
+        {
+            timestamp = fmt::format("[{:%Y-%m-%d %H:%M:%S}] ", fmt::localtime(std::time(nullptr)));
+        }
+
+        const int intendSize = getIntendSize();
+        switch (level)
+        {
+            case Level::info:
+                fmt::print(_file, "{}[INF] {:<{}}\n", timestamp, message, intendSize);
+                break;
+            case Level::warning:
+                fmt::print(_file, "{}[WARN] {:<{}}\n", timestamp, message, intendSize);
```

Comment:
```suggestion
                fmt::print(_file, "{}[WRN] {:<{}}\n", timestamp, message, intendSize);
```
It should be like this